src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -160,43 +160,15 @@ export const MarketWithdraw = (): JSX.Element => {
 
     let rateMultiplier = 1;
     let preferredCurrencyAmount = Big(1);
-    if (assetTicker.includes('L-BTC')) {
+    if (assetTicker === 'L-BTC' || assetTicker === 'tL-BTC') {
```

Comment:
You can write `assetTicker === LBTC_TICKER[network]`

Reply:
Done